### DISCUSSION TITLE
Will there be further notes???

### DISCUSSION THREAD

**Discussion by eal3019** [amroibrahim/DIYDoom#50] [General]
Hello Amro, What a great repository this is. I have always wanted to know how doom was programmed. Your explanations are very concise. I was wondering if you were going to continue with further notes (Notes 020 the end???? no way)

Thanks!

**Comment by amroibrahim** [comment]
Yes, once I get some time, I'm planing to re-write the full thing, also I'm planing to go into texturing, AI, menu system.

**Comment by a-westermann** [comment]
This has been really great to follow along and build in Python. Hoping to see more soon, if not I'll have to struggle along alone!
Thanks for creating this.

**Comment by Nille-hub** [comment]
Thank you for sharing this and for all the hard work you put into this (and I think there was a second coder helping in between?).

The culture regarding willingness to share knowledge and contribute within IT in general is something other sectors and people working outside this field should take as a guiding example. I am again and again amazed at the amount of effort people put into projects simply because of passion or kindness and then sharing it without a second thought

**Comment by rmi111** [comment]
I really wish at least you finished the rest of the tutorial.

**Reply by Nille-hub** [reply]
Never look a gift horse in the mouth ;)
